README.md

```diff
@@ -1,47 +1,117 @@
+
+There are a number of [open issues under discussion](https://github.com/opengeospatial/ogc-feat-geo-json/labels/waiting%20for%20input). 
 
-* The current draft version 0.1 will be the basis for testing JSON-FG by communities, in code sprints, etc.
-* We currently target an updated draft version 0.2 in mid 2023 to have a stable version for testing.
+Uptake in implementations and sufficient feedback from implementation experience are a prerequisite for this specification to eventually progress towards an OGC Standard.
 
 ## Overview
 
-JSON is a popular encoding format for geospatial data. The light weight, simple syntax, and clear human and machine readability of JSON appeals to developers. GeoJSON has become a very popular encoding and is supported in most deployments of APIs implementing OGC API Features, but has limitations that prevent or limit its use in some cases.
+[GeoJSON](https://www.rfc-editor.org/rfc/rfc7946.html) is a very popular encoding for geospatial vector data. GeoJSON is widely supported, including in most deployments of APIs implementing the OGC API Features Standard. However, GeoJSON has intentional restrictions that prevent or limit its use in certain geospatial application contexts. For example, GeoJSON is restricted to WGS 84 coordinates, does not support volumetric geometries and has no concept of classifying features according to their type.
+
+OGC Features and Geometries JSON (JSON-FG) is an OGC Candidate Standard for GeoJSON extensions that provide standard ways to support such requirements. The goal is to focus on capabilities that may require some geospatial expertise, but that are useful for many. Edge cases are considered out-of-scope of JSON-FG.
+
+Since JSON-FG specifies extensions to GeoJSON that conform to the GeoJSON standard, valid JSON-FG features or feature collections are also valid GeoJSON features or feature collections.
+
+The JSON-FG Standard specifies the following extensions to the GeoJSON format:
+
+* The ability to use Coordinate Reference Systems (CRSs) other than WGS84 (OGC:CRS84);
+* Allows the use of non-Euclidean metrics, in particular ellipsoidal metrics;
+* Support for solids and prisms as geometry types;
+* The ability to encode temporal characteristics of a feature; and
+* The ability to declare the type and the schema of a feature.
+
+Geographic features, their properties, and their spatial extents that can be represented as GeoJSON objects are encoded as GeoJSON. Additional information not specified in the GeoJSON RFC is mainly encoded in additional members of the GeoJSON objects. The additional members use keys that do not conflict with existing GeoJSON keys. This was done so that existing and future GeoJSON clients can continue to successfully parse and understand GeoJSON encoded content. JSON-FG enabled clients will also be able to parse and understand the additional members.
```

Comment:
"the GeoJSON RFC" > standards experts / spec writers will know what this means but others maybe not. Suggestions: replace with "the GeoJSON standard" or similar wording.

Reply:
changed as proposed